index.bs

```diff
@@ -1716,16 +1719,35 @@ associated.
 </xmp>
 <div dfn-type="dict-member" dfn-for="PublicKeyCredentialEntity">
     :   <dfn>name</dfn>
-    ::  A human-readable name for the entity. Its function depends on what the {{PublicKeyCredentialEntity}} represents:
+    ::  A [=human palatability|human-palatable=] name for the entity. Its function depends on what the {{PublicKeyCredentialEntity}} represents:
 
-          - When inherited by {{PublicKeyCredentialRpEntity}} it is a human-friendly identifier for the [=[RP]=], intended only
+          - When inherited by {{PublicKeyCredentialRpEntity}} it is a [=human palatability|human-palatable=] identifier for the [=[RP]=], intended only
             for display. For example, "ACME Corporation", "Wonderful Widgets, Inc." or "ОАО Примертех".
+
+              - [=[RPS]=] SHOULD perform enforcement, as prescribed in [[!RFC8266]]
+                Section 2.3 for the Nickname Profile of the PRECIS FreeformClass [[!RFC8264]],
```

Comment:
Change this and similar constructions from `[[!RFC8264] Section 2.3` to `Section 2.3 of [[!RFC8264]`.

Reply:
sounds fine to me. thx for review.